cpu/cc26x2_cc13x2/cpu.c

```diff
@@ -42,3 +42,11 @@ void cpu_init(void)
     /* trigger static peripheral initialization */
     periph_init();
 }
+
+#if CONFIG_CC26X2_CC13X2_FLASH_CCFG
+
+#define CCFG_FORCE_VDDR_HH (CONFIG_CC26X2_CC13X2_FORCE_VDDR_HH)
+
+#include "startup_files/ccfg.c"
```

Comment:
I agree with it, however, it stills needs to be included (the `startup_files/ccfg.c` file) to configure it as we need to define some preprocessor symbols to add the wanted configuration (vddr_hh, bootloader, DCDC, etc).

IMO it could be configured with Kconfig or through `CFLAGS`, or... should be better to supply a by default configuration as with `cc2538`?

Something like this I have in mind:

https://github.com/btcven/RIOT/blob/b4fc04b2cec006bef61d0771cd639bccfcdefffe/cpu/cc26x2_cc13x2/Kconfig#L50-L62

Reply:
Are those not in a header file that can be included in `ccfg.c`? It looks like you've modified that file anyway, might as well add another header then.

Reply:
I still see some issues with using a module approach, e.g.: defining the vector table offset configuration would require to know `cortex_vector_base` address, like `cc2538` does with their CCA (pretty similar).

Something like this comes to mind:

```c
extern cortexm_base_t cortex_vector_base;
#define SET_CCFG_IMAGE_VALID_CONF_IMAGE_VALID (uint32_t)&cortex_vector_base

#include "startup_files/ccfg.c"
```